pkg/cmd/secrets.go

```diff
@@ -159,6 +159,7 @@ JSON Secret: "{\"logging\": \"info\"}"`,
 func secrets(cmd *cobra.Command, args []string) {
 	jsonFlag := utils.OutputJSON
 	raw := utils.GetBoolFlag(cmd, "raw")
+	vistypes := utils.GetBoolFlag(cmd, "vistypes")
```

Comment:
Torn on whether to call this `--vistypes` or `--visibility`

Reply:
I struggled with this too. I settled on `vistypes` for consistency (e.g. the API deals with `rawVisType`, `computedVisType` for gets, `visType`, `originalVisType` for posts) and brevity. I think this is the right call; there's nothing else `visType` could be conflated with.

Reply:
Are you sold on `visType` vs switching to `visibility` throughout our API/CLI/etc? `visType` isn't much shorter than `visibility`, and I think `visibility` is more self-documenting than `visType`. `visType` makes more sense to me as an internal name, but not as a public-facing name.

Reply:
I'm very much on the fence, but I can get behind `visibility` for the public-facing name. Will make it so.